doc/source/dev/development_environment.rst

```diff
@@ -62,9 +62,10 @@ test and use your changes (in ``.py`` files), by simply restarting the
 interpreter.
 
 Note that another way to do an inplace build visible outside the repo base dir
-is with ``python setup.py develop``.  This doesn't work for NumPy, because
-NumPy builds don't use ``setuptools`` by default.  ``python setupegg.py
-develop`` will work though.
+is with ``python setup.py develop``.  The difference is that this instead of
```

Comment:
The only thing that that accomplishes is hiding the build log I think, which is horrible when you're developing. So no, didn't consider ``pip`` here.